src/settings.cpp

```diff
@@ -1057,14 +1065,27 @@ void Settings::overrideDefaults(Settings *other)
 		}
 		const FlagDesc *flagdesc = getFlagDescFallback(setting.first);
 		if (flagdesc) {
-			// Flags cannot be copied directly.
```

Comment:
while you're editing this method, is it possible to make the `Settings *other` argunment `const`?